app.js

```diff
@@ -2,10 +2,11 @@ const RickAndMortyAPI = 'https://rickandmortyapi.com/api/character/';
 let favoriteCharactersData = [];
 let characterGridData = [];
 let currentPage = 1;
+let currentPageData; 
 
 // ARM BUTTONS
 const searchInput = document.getElementById('filter-search');
-const favoriteBar = document.getElementById('favorite-container');
+const favoriteBar = document.getElementById('favorite-card-container');
```

Comment:
Created a new container for the favorite cards so that the 'Favorites' header isn't cleared once a favorite card is loaded